KiraCore/sekai
# V0.1.21.6 - modifications for code cleanup and small issues

- Update env to be latest
- Remove unused message type and commented codebase
- Check changes to make for codec registration
- Modification for codec meta registration
- Remove unused Proposer field from MsgApproveIdentityRecords
- Add title into the proposal
- For identity record creation/edit, update date to use blockTime
- Shell script modifications
